docs/content/doc/installation/from-binary.en-us.md

```diff
@@ -80,6 +83,13 @@ chmod 750 /etc/gitea
 chmod 644 /etc/gitea/app.ini
 ```
 
+### Configure Gitea's working directory
+
+**NOTE:** If you plan on running Gitea as a Linux service, you can skip this step as the service file allows you to set `WorkingDirectory`. Otherwise, consider setting this environment variable (semi-)permanently so that Gitea consistently uses the correct working directory.
+```
+export GITEA_WORK_DIR=/var/lib/gitea/
```

Comment:
This is by far the #1 confusion I've seen with new users in the past week or so hanging around in the chat and glad to see it here. Most of the confusion I've seen is that there is no direct example of running gitea directly from the command line so most people end up doing it wrong in one way or another, and perhaps if there were additionally a specific example of running it below this (after the suggestion of copying it to /usr/local/bin)
```
$ GITEA_WORK_DIR=/var/lib/gitea/ /usr/local/bin/gitea web -c /etc/gitea/app.ini
```
It could help avoid that all together.

Reply:
I added a section for running gitea now, does it seem to be clear enough?

Reply:
looks great!